### ISSUE TITLE
Atlas performance balance pass

### ISSUE THREAD

**Issue by zorg2044** [CobaltWolf/Bluedog-Design-Bureau#1456]
Need to verify updated Atlas parts are well balanced by testing a few key builds and seeing if the performance is sensible.

testing will be done is KSRSS reborn. (IRL orbit - ksp atmo height) x 0.25 + KSP atmo height

Some amount of excess performance for player margin is fine. Its reasonable to assume most of the IRL reference missions arent completely maxing out the performance of the rocket.

**Comment by zorg2044** [comment]
![image](https://github.com/CobaltWolf/Bluedog-Design-Bureau/assets/39182212/d4169c91-b800-41b5-911b-fd2352d62f4f)

Haven't tested everything in the list but in general I feel like I'm not going to make any changes to the fuel loads. The current base tank has the same prop load as the old build to the same height. All the extension tanks are based on the formula we use for volume. 

Due to various factors the performance of KSP rockets can vary a lot but I consider between 2/3 to 100% of IRL performance in 2.5x scale to be acceptable.

The IRL figures here are from example missions from Ed Kyle so are indicative rather than max payload. From Atlas 2 onwards the figures are max payload from Lockheed/ULA users guides.

Atlas II struggles to LEO but this is due to the difference in scaling between the world vs the atmo I believe. It suffers significant gravity losses and the core never gets close to 1:1 TWR. However most other builds fall in an acceptable range so I'm planning to leave the balance as is. Some small tweaks to structural parts mass might still happen.

**Comment by JacobB094** [comment]
I agree, I've been flying with WIP variants, and they're wonderful. Atlas II struggles to lower orbits, but it's not really meant for that.  In fact, through Atlas-Centaur's entire history, before Atlas V there wasn't a single LEO launch since they finished testing the Centaur. So those numbers were purely theoretical in first place.

I even recreated the triple barrel Atlas from the brochure, and it turned out to be a surprisingly competent lifter. So I say, the current balance on Atlas is close to perfect. It plays really nice with other parts, too.

**Comment by JacobB094** [comment]
Further testing revealed one area where balance could be improved: Atlas V SRBs. It's less the payload, which is fine, and more their thrust. At its LEO maximum of 16T of payload, Atlas V 552 has a launch TWR of whoopin' 1.81. That is a little much. Likewise, an Atlas V 422 (similar to Starliner lifter) has a TWR of 1.52 with a 14.4T payload. 

Payload numbers look great, but TWR is excessive on every Atlas V that uses boosters of either type, even with maximum payload. Could this be looked at? All other solid-augmented Atlases check out and fly nicely.

**Comment by zorg2044** [comment]
> Further testing revealed one area where balance could be improved: Atlas V SRBs. It's less the payload, which is fine, and more their thrust. At its LEO maximum of 16T of payload, Atlas V 552 has a launch TWR of whoopin' 1.81. That is a little much. Likewise, an Atlas V 422 (similar to Starliner lifter) has a TWR of 1.52 with a 14.4T payload.
> 
> Payload numbers look great, but TWR is excessive on every Atlas V that uses boosters of either type, even with maximum payload. Could this be looked at? All other solid-augmented Atlases check out and fly nicely.

We have a strict formula we follow for all solids to convert from IRL thrust and propellant mass. I'm not inclined to arbitrarily tweak thrust. Its been given a steady drop thrust curve however.

**Comment by JacobB094** [comment]
OK, I guess. In fact, I just checked the PPG for Atlas V, and wouldn't you know it, the 5 booster version _does_ have about 1.8G of acceleration at launch, and only goes up from there. So I guess it's weird, but not wrong. I did not expect a modern launch vehicle to go up quite that fast, but it doesn't actually impact its flyability, at least the way I ran the tests. The curve will no doubt help tame the burnout acceleration a bit, at least.
